src/sentry/static/sentry/app/components/actions/resolve.jsx

```diff
@@ -184,7 +185,10 @@ export default class ResolveActions extends React.Component {
                       }}
                     >
                       {latestRelease
-                        ? t('The current release (%s)', latestRelease.version)
+                        ? t(
+                            'The current release (%s)',
+                            <Version version={latestRelease.version} anchor={false} />
```

Comment:
I am not so familiar with our translation tools yet, but please, is it fine to have a 'non-string' in here?

Reply:
It worked fine, but I have changed it to function that returns string just to be sure.